### DISCUSSION TITLE
Badges - display an empty dot instead of the exact number of notifications

### DISCUSSION THREAD

**Discussion by novellizator** [invertase/react-native-firebase#6239] [Q&A] [answered]
So notifications can have a badge appended and that is displayed as small dot with a number inside.
Since it's too much code on the backend to give a proper count, I'd like to stick to an empty dot for "there are notifications" vs no dot when there are none.

So probably the best would be to be able to reinterpret any nonzero number as an empty dot. Is that possible?

**Comment by mikehardy** [comment, marked as answer]
I assume this is on iOS? I don't think there are badges on android ? https://notifee.app/react-native/docs/ios/badges

At any rate, the native APIs provide no ability to do this - they offer a value of "0" which will clear the badge, or a number, there's no option I'm aware of to just get a "there's something new" dot

https://developer.apple.com/documentation/uikit/uiapplication/1622918-applicationiconbadgenumber?language=objc
